docs/docusaurus/docs/cloud/setup/setup_cloud.md

```diff
@@ -0,0 +1,183 @@
+
+    - **Token name** - Enter a name for the token that will help you quickly identify it.
+
+    - **Role** - Select **Admin**. For more information about the available roles, click **Information** (?) .
+
+4. Click **Create**.
+
+5. Copy the access token and store it in a secure location. The token can't be retrieved after you close the dialog.
+
+6. Click **Close**.
+
+7. Copy the value in the **Organization ID** field and save it in the same location with your user access token.
+
+
+## Create a Data Context
+
+A Data Context provides the configurations and methods for GX Cloud components, including Data Sources, Expectations, Profilers, and Checkpoints.
+
+1. In Jupyter Notebook, copy the following code into a cell to instantiate the GX Cloud Data Context.
+
+    ```python title="Jupyter Notebook"
+    os.environ["GX_CLOUD_ACCESS_TOKEN"] = "<user_access_token>"
+    os.environ["GX_CLOUD_ORGANIZATION_ID"] = "<organization_id>"
+    context = gx.get_context()
+    ```
+2. Replace `user_access_token` and `organization_id` with the values you created and saved previously. See [Generate your user access token and copy your organization ID](#generate-your-user-access-token-and-copy-your-organization-id).
+
+3. Run the code.
+
+## Connect to a Data Source
+
+A Data Source provides a standard API for accessing and interacting with data from a data source system.
+
+In Jupyter Notebook, run the following code to connect to existing `.csv` NYC taxi trip data stored in the `great_expectations` GitHub repository:
+
+```python title="Jupyter Notebook"
+datasource_name = "Test"
+datasource = context.sources.add_pandas(datasource_name)
+asset_name = "Test"
+path_to_data = "https://raw.githubusercontent.com/great-expectations/gx_tutorials/main/data/yellow_tripdata_sample_2019-01.csv"
+asset = datasource.add_csv_asset(asset_name, filepath_or_buffer=path_to_data)
+batch_request = asset.build_batch_request()
+```
+
+## Create an Expectation Suite
+
+An Expectation Suite is a collection of verifiable assertions about data. 
+
+In Jupyter Notebook, run the following code to create a new, empty Expectation Suite:
+
+```python title="Jupyter Notebook"
+expectation_suite_name = "Test"
+expectation_suite = context.add_expectation_suite(
+    expectation_suite_name=expectation_suite_name
+)
+```
+
+## Add an Expectation to an Expectation Suite
+
+An Expectation is a verifiable assertion about data. They take implicit assumptions about your data and make them explicit.
+
+1. In Jupyter Notebook, run the following code to add an Expectation to the **Test** Expectation Suite and display the Expectation settings:
+
+    ```python title="Jupyter Notebook"
+    expectation_suite_id = expectation_suite.ge_cloud_id
+    expectation_suite = context.get_expectation_suite(ge_cloud_id=expectation_suite_id)
+    column_name = "vendor_id"
+    expectation_configuration = gx.core.ExpectationConfiguration(**{
+    "expectation_type": "expect_column_min_to_be_between",
```

Comment:
I'd suggest we include at least one more Expectation - a simple one (for example, expect_column_values_to_not_be_null) that passes. Then users will be able to see both a passing and failing Expectation.

Reply:
@rachhouse the existing Expectation passes:

![image](https://github.com/great-expectations/great_expectations/assets/104205257/52fdc042-bf29-49db-803b-68536ec90651)

We could add an Expectation that fails, but we would need to explain why it failed and why it is significant. This would lengthen the procedure. I'm not opposed to the idea and welcome your recommendations for providing an Expectation that fails, the reason it failed, and why a failure is helpful.

Reply:
My bad, my brain was a bit fried after support when I reviewed - I still had our conversation about negative fares in my head. I think just having one passing Expectation is ok if we don't want to have the series of steps for "look at the failing Validation, fix the failing Expectation, look at the history of validation runs." 

However, I think we could pick a more immediately relatable Expectation for our passing example. Using `expect_column_min_to_be_between` and then only supplying a `min_value` (instead of a min and max, since it is a between expectation) could be weird for a new user. Additionally, expecting the minimum `vendor_id` to be greater than `0.1` doesn't seem like a real-world case - at most, we'd probably care that the `vendor_id` field is not null, and possibly that it matches a particular pattern (for example, numeric or alphanumeric).

A couple simple expectations that I'd have of this dataset are that:
- `pickup_datetime` is not null
- `passenger_count` is greater than zero
-  `dropoff_datetime` is greater than `pickup_datetime`